app/build.gradle

```diff
@@ -51,6 +55,7 @@ dependencies {
     compile 'com.android.support:design:25.3.1'
     compile 'com.google.code.gson:gson:2.7'
     compile 'org.greenrobot:eventbus:3.0.0'
+    compile 'cn.pedant.sweetalert:library:1.3'
```

Comment:
Yes, and I've even build a popup myself. But then I encountered this library, which makes it much easier and cleaner to create popups